### DISCUSSION TITLE
Enhanced Workflow | What will happen to existing projects starting from v14?

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#570] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

v13

### Description

Hi guys,
in the "Project Creation" page I see the following infobox:
<img width="999" alt="Screenshot 2024-10-25 at 14 57 47" src="https://github.com/user-attachments/assets/576dc328-761d-4462-bf2f-e02d0c0c45ea">

The infobox talks about "Project creation", so the existing projects will not be converted to the Enhanced Workflow, is that right?

Thanks.

**Comment by fredmaggiowski** [comment, marked as answer]
Hi @silversoul93, existing Projects will keep on working as usual. The only thing that is going to change with v14 is that it will only be possible to create new projects using the Enhanced.

We won't oblige projects to migrate to the enhanced workflow yet, however it may become required at a certain point in the future (maybe v15? there is no defined roadmap for this yet).

**Reply by frengooo** [reply]
Thanks @fredmaggiowski 😄
